projects/plugins/jetpack/modules/shortcodes/js/slideshow-shortcode.js

```diff
@@ -126,9 +126,9 @@ JetpackSlideshow.prototype.finishInit_ = function () {
 				}
 			}
 		} );
-	} else {
-		this.element.children( ':first' ).show();
-		this.element.css( 'position', 'relative' );
+	} else if ( this.element.children.length ) {
+		this.element.children[ 0 ].style.display = 'block';
+		this.element.style.position = 'relative';
```

Comment:
```suggestion
		jQuery( this.element ).children( ':first' ).show();
		jQuery( this.element ).css( 'position', 'relative' );
```
I tested your solution and it works. Just wondering why you didn't wrap it in jQuery, as other elements in the code are (eg.` jQuery( this.element ).cycle` on line 99)

Reply:
I just don't feel comfortable introducing new jQuery code if we can avoid it.

Reply:
Yeah, makes sense 👍